sphinx + rst2pdf error: unexpected keyword argument 'ts'

### Error while trying to generate the pdf with rst2pdf

### Problem
I was using sphinx and rst2pdf without any problems but the issue occured today upon a fresh installation on another PC. I'm no longer able to generate the documentation in pdf format. The only difference between my configuration and the new one is the sphinx version: I have v1.7.4 while the new pc has v.1.7.6 (installed by default with pip install command).

#### Procedure to reproduce the problem
```
sphinx-build -b pdf source build/pdf
```

#### Error logs / results
```
writing MyDocsTryOut... [ERROR] pdfbuilder.py:130 __init__() got an unexpected keyword argument 'ts'
Traceback (most recent call last):
  File "c:\python27\lib\site-packages\rst2pdf\pdfbuilder.py", line 127, in write
    docwriter.write(doctree, destination)
  File "c:\python27\lib\site-packages\docutils\writers\__init__.py", line 80, in write
    self.translate()
  File "c:\python27\lib\site-packages\rst2pdf\pdfbuilder.py", line 621, in translate
    compressed=self.compressed)
  File "c:\python27\lib\site-packages\rst2pdf\createpdf.py", line 653, in createPdf
    pdfdoc.multiBuild(elements)
  File "c:\python27\lib\site-packages\reportlab\platypus\doctemplate.py", line 1135, in multiBuild
    self.canv.save()
  File "c:\python27\lib\site-packages\reportlab\pdfgen\canvas.py", line 1240, in save
    self._doc.SaveToFile(self._filename, self)
  File "c:\python27\lib\site-packages\reportlab\pdfbase\pdfdoc.py", line 215, in SaveToFile
    data = self.GetPDFData(canvas)
  File "c:\python27\lib\site-packages\reportlab\pdfbase\pdfdoc.py", line 241, in GetPDFData
    return self.format()
  File "c:\python27\lib\site-packages\reportlab\pdfbase\pdfdoc.py", line 418, in format
    IOf = IO.format(self)
  File "c:\python27\lib\site-packages\reportlab\pdfbase\pdfdoc.py", line 866, in format
    fcontent = format(self.content, document, toplevel=1)   # yes this is at top level
  File "c:\python27\lib\site-packages\reportlab\pdfbase\pdfdoc.py", line 80, in format
    f = element.format(document)
  File "c:\python27\lib\site-packages\reportlab\pdfbase\pdfdoc.py", line 1546, in format
    D['ModDate'] = D["CreationDate"] = PDFDate(ts=document._timeStamp,dateFormatter=self._dateFormatter)
TypeError: __init__() got an unexpected keyword argument 'ts'
```

### Environment info
- OS: Windows 10
- Python version: 2.7.15
- Sphinx version:  1.7.6

--- a/CHANGES.rst
+++ b/CHANGES.rst
@@ -16,6 +16,7 @@
 * Fixed: ``:linenos_offset:`` now works again
 * Fixed: Removed debugging print statement when using line blocks
 * Fixed: Removed uniconverter from setup (issue 487)
+* Fixed: Error when using --date-invariant with newer reportlab versions (Issue 678)
 
 
 0.93 (2012-12-18)

--- a/rst2pdf/createpdf.py
+++ b/rst2pdf/createpdf.py
@@ -1521,7 +1521,7 @@ def patch_PDFDate():
     class PDFDate(pdfdoc.PDFObject):
         __PDFObject__ = True
         # gmt offset now suppported
-        def __init__(self, invariant=True, dateFormatter=None):
+        def __init__(self, invariant=True, ts=None, dateFormatter=None):
             now = (2000,01,01,00,00,00,0)
             self.date = now[:6]
             self.dateFormatter = dateFormatter

